### ISSUE TITLE
Add Letter Generation Flow Editabilty

### ISSUE THREAD

**Issue by jeanwatts** [PublicDataWorks/complaint-manager#220]
### Business Context:
Blocked By #219 
Hawaii will want the ability to have different letter generation flows for different letter types to support their many different kinds of letter generation needs.  This story also contributes to the modularization of the app for future org support. 

**As a** Admin User
**I want** to be able to customize the flow of my letter generation
**So that** My letter generation process is easier and more aligned with each of my individual letter types.

### In Scope
1. Modify the Edit button in the Letters section of the Admin page to be a "button of buttons" with the options "Letter Type"  and "Letter Generation Steps" 
2. "Letter Type" should route to existing Edit letter types page
3. "Letter Generation Steps" should bring up a Modal where the user can change what steps their generation process has and their order
4. Create handler to PUT input pages for a given letter type for letter type Letter input Pages
Note: A user can select anywhere from No Steps to all of them
### Out of Scope
1. Changing generation flow based on inputs
2. Adding new possible steps to generation process
3. Pact testing this story

### Open Questions
1.
2.
3.

## Acceptance Criteria

1. Scenario: 
Given
When
Then

2. Scenario: 
Given
When
Then

### Checklist (should be reviewed before grooming as the BA or Pair are writing the card—NOT reviewed for the first time at grooming)
0. Discuss the technical implementation of this card.
1. Discuss how this feature would be included in or affect the Audit Log.
2. In respect to security, what could go wrong? Brainstorm threats using STRIDE model, especially if this card introduces a new technical architecture or design pattern. (TIMEBOX: 7 mins) https://docs.google.com/document/d/18AFl1d3SRieI4d_8R3GyEazqQOycLNDHGN_vOYIHuiE/edit#heading=h.aio9zb5j63wl
3. Ensure any open questions have been resolved with stakeholders.
5. Determine if documentation should be referenced or created.
6. Ensure to include all necessary mockups to this card.
7. Discuss what success/failure user feedback messages would be present, if any occur (front end/back end).
8. Discuss why this feature would not be implemented behind a toggle / what would happen if this card was partially done while a release branch was being cut.
9. If this card is part of a sequence of cards, discuss if this card should be blocked until the testing of previous  cards in the sequence have been finished.
10. Have we added edge cases as ACs that we're considered from a QA perspective
11. Does this card overlap with any entities outlined in [API Housekeeping](https://www.pivotaltracker.com/story/show/173963545)?  

**Notes:**
